src/Shared/TokenGenerator.cs

```diff
@@ -0,0 +1,38 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Security.Cryptography;
+
+namespace Aspire.Hosting;
+
+internal static class TokenGenerator
+{
+    public static string GetToken()
```

Comment:
`GetToken` makes it seem like you might get the same value if you called the method multiple times. Consider `CreateToken` or similar.